src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/Demo/ChangeColor.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.SpectatorView
+{
+    public class ChangeColor : MonoBehaviour
```

Comment:
>ChangeColor [](start = 17, length = 11)

Is it necessary for this to be a public class in the main project? If so, I think it needs some documentation about what it's for.

Reply:
no, this can be internal